src/ReverseProxy/Service/Management/ProxyConfigManager.cs

```diff
@@ -311,6 +313,13 @@ private void UpdateRuntimeClusters(IList<Cluster> newClusters)
                                     settings: newCluster.SessionAffinity?.Settings as IReadOnlyDictionary<string, string>),
                                 httpClient,
                                 newClusterHttpClientOptions,
+                                new ClusterProxyHttpRequestOptions(
+                                    requestTimeout: newCluster.HttpRequest?.RequestTimeout ?? TimeSpan.FromSeconds(100),
+                                    version: newCluster.HttpRequest?.Version ?? HttpVersion.Version20
+#if NET
+                                    , versionPolicy: newCluster.HttpRequest?.VersionPolicy ?? HttpVersionPolicy.RequestVersionOrLower
+#endif
```

Comment:
Pass the nulls through, let the consumer choose the default if it's not set.
```suggestion
                                    requestTimeout: newCluster.HttpRequest?.RequestTimeout,
                                    version: newCluster.HttpRequest?.Version
#if NET
                                    , versionPolicy: newCluster.HttpRequest?.VersionPolicy
#endif
```

Reply:
fixed.